packages/web-component/src/lib/helpers/conditions.ts

```diff
@@ -9,14 +11,16 @@ export const conditions: ConditionsMap<Context> = {
     'is-true': (ctx) => !!ctx.code,
     'is-false': (ctx) => !ctx.code,
   },
+  [elseInteractionId]: {
+    // always true
+  },
```

Comment:
WDYT about returning a function like the other conditions?
I think it will be more consistent

Reply:
function of what operator?

Reply:
Without an operator
something like
```
[elseInteractionId]: () => true,
```

Reply:
So we need to change the def of the conditionsMap